src/constraints.jl

```diff
@@ -26,3 +26,14 @@ function satisfy_constraint(constraint::PopulationConstraint,
     end
     return false
 end
+
+function satisfy_constraint(constraint::PopulationConstraint,
+                            D₁_pop::Int,
+                            D₂_pop::Int)
+    if D₁_pop >= constraint.min_pop && D₁_pop <= constraint.max_pop
+        if D₂_pop >= constraint.min_pop && D₂_pop <= constraint.max_pop
+            return true
+        end
+    end
+    return false
+end
```

Comment:
Previously had a satisfy_constraint(constraint, proposal) format, but having to build a proposal before the constraint slowed things up so I put this up here for now. The previous one is still up there.